app/static/playlist.css

```
@@ -199,3 +199,33 @@ p:first-child {
   background: #000;
   color: #fff;
 }
+
+.indigenous {
+  position: fixed;
+  bottom: 20px;
+  right: 0;
+  padding: 0 75px;
+  display: none;
+  width: 685px;
```

```suggestion
  width: 100%;
```
If you set width to 100% here you won't need another media query for the 720px max-width (and it'll render nicely in Safari :-) )